include/robots/ackermann/ackermann_base.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+namespace BoBRobotics {
+namespace Robots {
+namespace Ackermann {
+//----------------------------------------------------------------------------
+// BoBRobotics::Robots::AckermannBase
+//----------------------------------------------------------------------------
+//! A base class for robots with Ackermann-type steering
+template<class Derived>
```

Comment:
What's the purpose of this base class? Couldn't you remove it altogether? At first I thought it might be for ``std::enable_if``` but it doesn't seem to be

Reply:
Nothing, really. I left it as a placeholder but we could remove it entirely... unless we use it for type traits (see below)